docs/ref/request-response.txt

```diff
@@ -135,7 +135,9 @@ All attributes should be considered read-only, unless stated otherwise.
 
 .. attribute:: HttpRequest.META
 
-    A dictionary containing all available HTTP headers. Available headers
+    A dictionary containing the `WSGI environ \
```

Comment:
This change looks unrelated to the patch and should be submitted separately.

Reply:
I think that removing this and submitting it separately would lack the context later; what I did was just improving the docs a little bit more. From my experience, newcomers feel extremely confused when they find the prefix `HTTP_`.

Reply:
Just the PR is fine. 🙂